picard/acoustid/json_helpers.py

```diff
@@ -34,16 +34,48 @@ def _make_releases_node(recording):
                 release_mb['title'] = release['title']
             else:
                 release_mb['title'] = release_group['title']
+
             if 'country' in release:
                 release_mb['country'] = release['country']
+
+            if 'date' in release:
+                release_mb['date'] = release['date']
+
+            if 'medium_count' in release:
+                release_mb['medium_count'] = release['medium_count']
+
+            if 'track_count' in release:
+                release_mb['track-count'] = release['track_count']
+
             release_mb['media'] = []
             for medium in release['mediums']:
                 medium_mb = {}
                 if 'format' in medium:
                     medium_mb['format'] = medium['format']
-                medium_mb['track-count'] = medium['track_count']
+
+                if 'track_count' in medium:
+                    medium_mb['track-count'] = medium['track_count']
+
+                if 'position' in medium:
+                    medium_mb['position'] = medium['position']
+
+                if 'tracks' in medium:
+                    medium_mb['tracks'] = medium['tracks'][0]
```

Comment:
What exactl is `medium['tracks'][0]` here? Isn't this just the first track? Would it have the correct format?

Reply:
element zero because the json structure supports a list, but we did a search on single track, so would never have more than one Track element, so no looping needed.  I'll add comment to code.

Reply:
But `medium_mb['tracks']` should have the same format in both cases. If it is the list in one case and a single object in the other it makes it difficult to use.

There is also the risk that code uses `len(medium_mb['tracks'])` to get the track count, but I don't think we have this.

Reply:
ok, I'm guessing from one of your other recent comment, that by "both cases" you mean musicbrainz vs acoustId, so now I understand that "json_helpers" is a converter, and result should look like MB.  I will make them have same structure for tracks, and also add a comment to code about this.

Reply:
Ah, yes. Sorry if this is not clear. Basically all the code run against this result later (mostly the matching in `Metadata`) is using the structure as returned by the MB web service. So in order to use the same logic for the AcoustID results the `parse_recording` function is supposed to rewrite the data from AcoustID into the same structure.

You can consider the result of this function the internal representation of the data as Picard uses it. But to simplify things this matches the structure as it is returned by MB web service. AcoustID data is similar enough so it can be rewritten.